test/parallel/test-assert.js

```diff
@@ -409,6 +409,14 @@ assert.throws(() => {
 }, /Got unwanted exception: user message/,
               'a.doesNotThrow ignores user message');
 
+assert.throws(() => {
```

Comment:
@refack thanks for the feedback. `throws` calls `innerThrows` internally and `doesNotThrow` calls `innerThrows` internally as well. That's why we use function to test itself? Did I understand you correctly? Do we need just usual `try / catch` instead of `assert.throws` right?